src/channel_request/utils.rs

```
@@ -0,0 +1,8 @@
+
+// Need to add more conditions
+
+pub fn check_if_valid(value: u128, upper: u128, lower: u128) -> () {
```

This should probably return an error or a bool and needs a much more descriptive name. Also, I'm not sure if this really warrants a util function and wouldn't be more readable if just inlined.